pl_examples/hydra_examples/conf/data/fashionmnist.yaml

```diff
@@ -0,0 +1,32 @@
+# @package _group_
+
+ds:
+  cls: torchvision.datasets.KMNIST
+  params:
+    root: ./datasets
```

Comment:
remember that the working directory is changed by Hydra. is this not causing an issue here?

Reply:
Fixed